Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -187,10 +187,10 @@ If not installing with Helm, here are the steps to install with manifests.
    the KUBECONTEXT environment variable.
 
    ```console
-   export KUBECONTEXT=<cluster-name>
+   export KUBECONTEXT=hub-cluster
    ```
 
-1. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.
+1. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/external/import-external-cluster.sh) script.
```

Comment:
Based on [this comment](https://github.com/rook/rook/pull/13932#discussion_r1565242427), I thought additional parameters or settings were needed?